src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -166,7 +167,30 @@
 
 <bean id="FlatFileObjectSpecification" abstract="true" class="org.kuali.kfs.sys.batch.FlatFilePrefixObjectSpecification" />
 
-	
+	<bean id="kimFeedJob" parent="unscheduledJobDescriptor">
+		<property name="steps">
+			<list>
+				<ref bean="runKimFeedStep"/>
+			</list>
+		</property>
+	</bean>
+
+	<bean id="runKimFeedStep" class="edu.cornell.kfs.sys.batch.RunKimFeedStep" parent="step">
+		<property name="kimFeedProperties">
+			<props>
+				<prop key="db-url0">${edw.datasource.url}</prop>
+				<prop key="db-username0">${edw.datasource.username}</prop>
+				<prop key="db-password0">${edw.datasource.password}</prop>
+				<prop key="db-url1">${rice.datasource.url}</prop>
+				<prop key="db-username1">${rice.datasource.username}</prop>
+				<prop key="db-password1">${rice.datasource.password}</prop>
+				<prop key="skip-delta-flag-updates">${kim.feed.skip.delta.flag.updates}</prop>
```

Comment:
I am wondering if we should move kim.feed.skip.delta.flag.updates, kim.feed.load.latest.delta.only and kim.feed.load.delta.with.date to KFS parameters.  It makes total sense for the database connection information to be controlled from a properties file.  I know there is a seperate user story to refactor the job into our standards, but it may make sense to move these now since they affect properties files that will need to be added to each developer's system, and to the KFS application servers.

Reply:
I was hoping to keep these changes to a minimum, but I agree that moving the non-datasource-credential properties to parameters would be good to do now, which should also make it easier to perform testing (and make it easier for the DevOps config changes as well). I'll update this accordingly. Thanks!